packages/image_picker/image_picker/example/ios/Podfile

```diff
@@ -31,7 +31,10 @@ target 'Runner' do
   flutter_install_all_ios_pods File.dirname(File.realpath(__FILE__))
 
   target 'RunnerUITests' do
+    platform :ios, '9.0'
     inherit! :search_paths
+    # Pods for testing
+    pod 'OCMock', '~> 3.8.1'
```

Comment:
The target of the unit tests have been changed to `RunnerTests` recently, so this needs to be updated.

Reply:
Done.